proto/stream_service.proto

```diff
@@ -52,12 +52,18 @@ message DropActorsResponse {
   common.Status status = 2;
 }
 
+message PartialGraphInfo {
+  repeated uint32 actor_ids_to_send = 1;
+  repeated uint32 actor_ids_to_collect = 2;
+  repeated uint32 table_ids_to_sync = 3;
+}
+
 message InjectBarrierRequest {
   string request_id = 1;
   stream_plan.Barrier barrier = 2;
-  repeated uint32 actor_ids_to_send = 3;
-  repeated uint32 actor_ids_to_collect = 4;
-  repeated uint32 table_ids_to_sync = 5;
+  reserved 3, 4, 5;
```

Comment:
These are internal APIs right? I think we don't need to care about backwards compat here.